base/utils/R/convert.input.R

```diff
@@ -618,6 +618,18 @@ convert.input <- function(input.id, outfolder, formatname, mimetype, site.id, st
   }
   
   #--------------------------------------------------------------------------------------------------#
+  # Check if result has empty or missing files
+  
+  result_sizes <- result %>% purrr::map(~ .x$file) %>% purrr::map(~ .x %>% file.size) %>% as.data.frame() %>% 
+    `colnames<-`(c("file_size")) %>%
+    mutate(missing = ifelse(is.na(file_size),NA,TRUE)) %>%
+    mutate( empty = ifelse(file_size==0,"EMPTY",TRUE))
+  
+  if( NA %in% result_sizes$missing | "EMTPY" %in% result_sizes$empty){
```

Comment:
Related to my above comment, by using logical columns, this can be simplified to:

```
if (any(result_sizes$missing) || any(result_sizes$empty)) {...}
```

Also, you _almost never_ want to use single-character forms of `&` and `|` in `if/else` blocks, because `if` and `else` conditions are expected to be scalar whereas single character`&` and `|` are vectorized and can return a vector. `if/else` will take only the first item of vector conditions with a warning, which is almost never what you want.

If you have vector conditions that you're testing, you'll probably want to use them in combination with either `any(<condition>)` or `all(<condition>)`, and if you have multiple scalar conditions, you'll want to use the double-character forms (`&&` and `||`), which evaluate once left to right and evaluate as early as possible.

To see what I mean, compare the output of the following:

```
x <- c(TRUE, TRUE, FALSE)
y <- c(TRUE, FALSE, FALSE)
x | y # Result is length 3 vector
x || y # Result is SCALAR -- performs only the first comparison
if (x | y) print ("yes") # Should throw a warning about using only the first element
if (any(x | y)) print("yes") # Should print "yes"
if (all(x | y)) print("yes") # Should not print anything
```